docs/tutorials/excel-tutorial.md

```diff
@@ -229,23 +231,23 @@ Now that we know how to read and write to a single cell, let's generalize the sc
     Note that the array defining cell locations is zero-based. That means cell **A1** is `range[0][0]`.
 
     ```TypeScript
-      // Get the values of the used range.
-      let range = selectedSheet.getUsedRange();
-      range.load("rowCount,values");
-      await context.sync();
-
-      // Iterate over the fourth and fifth columns and set their values to their absolute value.
-      for (let i = 1; i < range.rowCount; i++) {
-        // The column at index 3 is column "4" in the worksheet.
-        if (range.values[i][3] != 0) {
-          selectedSheet.getCell(i,3).values = [[Math.abs(range.values[i][3])]];
-        }
-
-        // The column at index 4 is column "5" in the worksheet.
-        if (range.values[i][4] != 0) {
-          selectedSheet.getCell(i,4).values = [[Math.abs(range.values[i][4])]];
-        }
+    // Get the values of the used range.
+    let range = selectedSheet.getUsedRange();
+    range.load("rowCount,values");
```

Comment:
Is this the preferred notation, or range.load(["rowCount", "values"]), or do we not care?

Reply:
I don't think we have a standard here. FWIW, I think the array is more complicated.